plugins/woocommerce-blocks/assets/js/blocks/featured-items/register.tsx

```diff
@@ -68,6 +68,7 @@ export function register(
 		},
 		supports: {
 			...metadata.supports,
+			ariaLabel: true,
```

Comment:
What do you think if we remove `ariaLabel` from here and we add it in the respective `block.json` files:
- https://github.com/woocommerce/woocommerce/blob/5694da606815499eda60dc1eb3f8007b5d72efab/plugins/woocommerce-blocks/assets/js/blocks/featured-items/featured-category/block.json
- https://github.com/woocommerce/woocommerce/blob/5694da606815499eda60dc1eb3f8007b5d72efab/plugins/woocommerce-blocks/assets/js/blocks/featured-items/featured-product/block.json


This allows us to generate an updated [`block-reference.md` file](https://github.com/woocommerce/woocommerce/blob/5694da606815499eda60dc1eb3f8007b5d72efab/docs/building-a-woo-store/block-references.md#L8)

Reply:
Good callout, @gigitux 

Please review this commit: [Add suports.ariaLabel to Featured Category/Products block.json](https://github.com/woocommerce/woocommerce/pull/55351/commits/910ed00413c660cc4d9b789cc14cedd5b4a2b274)

cc: @dkotter